### DISCUSSION TITLE
KaTeX is not rendering TeX code, what's wrong? (sorry if it's a stupid question)

### DISCUSSION THREAD

**Discussion by michalmicpaw** [KaTeX/KaTeX#2627] [Q&A] [answered]
Hi.

I have such code:

https://paste.ofcode.org/fbTuT7TTFNWXN3MwT5R2tx

I've placed directory **katex** into main catalogue of page. But **katex** is not rendering my code. What am I doing wrong?

Thanks.
Mike

**Comment by edemaine** [comment, marked as answer]
You're not calling the autorender tool. Could you try following the directions on https://katex.org/docs/autorender.html and let us know if anything is unclear?

**Comment by michalmicpaw** [comment]
Thank you for quick aswer. I've added line with **onload** but i get error log:

`auto-render.min.js:1 Uncaught Error: No element provided to render
    at t.default (auto-render.min.js:1)`

https://paste.ofcode.org/9nGLPHqTfUnu7XHNug6xkq

PS.
OK it works, I just needed to put the code between '$$' and '$$'

**Comment by edemaine** [comment]
Look at the directions under `delimiters`, last example, if you want `$` to work.
